akka-cluster-typed/src/test/scala/akka/cluster/ddata/typed/scaladsl/ReplicatorSpec.scala

```diff
@@ -4,17 +4,14 @@
 
 package akka.cluster.ddata.typed.scaladsl
 
-import scala.util.Failure
-import scala.util.Success
+// FIXME move to doc package
```

Comment:
reminder

Reply:
let me do that in a separate PR, I'll add it to my my todo list